dkist/io/asdf/converters/models.py

```diff
@@ -60,8 +65,12 @@ def from_yaml_tree_transform(self, node, tag, ctx):
         )
 
     def to_yaml_tree_transform(self, model, tag, ctx):
+        crpix_key = "crpix_table"
+        # for asdf standard < 1.6
+        if tag.endswith("varying_celestial_transform-1.1.0"):
+            crpix_key = "crpix"
```

Comment:
@braingram how terrible is this? lol

The newer tags have a minimum standard version of 1.6, so our oldest supported asdf writes the old tag.